docs/stdlib.md

```diff
@@ -1210,6 +1172,7 @@ Removes a watchpoint
 | --- | --- |
 | [`assert`](#assert) | Simple assertion macro that will exit the entire script with an error code if the condition is not met. |
 | [`ppid`](#ppid) | Get the pid of the parent process |
+| [`socket_cookie`](#socket_cookie) | Retrieve the cookie (generated by the kernel) of the socket. |
```

Comment:
@jordalgo This is the "summary" that gets included in the table (the first line)

Without that parsing change, this table was quite monstrous ;)